build.gradle

```diff
@@ -1,81 +1,75 @@
 plugins {
-    id "com.jfrog.bintray" version "1.1"
-    id "com.github.jruby-gradle.base" version "0.1.5"
     id "java"
+    id "maven-publish"
+    id "org.embulk.embulk-plugins" version "0.4.2"
 }
-import com.github.jrubygradle.JRubyExec
 repositories {
     mavenCentral()
-    jcenter()
 }
-configurations {
-    provided
+
+embulkPlugin {
+    mainClass = "org.embulk.parser.NoneParserPlugin"
+    category = "parser"
+    type = "none"
 }
 
+group = "io.github.sonots"
 version = "0.2.0"
+description = "Embulk parser plugin not to parse at all."
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
 
 dependencies {
-    compile  "org.embulk:embulk-core:0.6.18"
-    provided "org.embulk:embulk-core:0.6.18"
+    compileOnly "org.embulk:embulk-api:0.10.31"
+    compileOnly "org.embulk:embulk-spi:0.10.31"
+    compile("org.embulk:embulk-util-config:0.2.1") {
+        // They conflict with embulk-core. They are once excluded here,
+        // and added explicitly with versions exactly the same with embulk-core:0.10.19.
+        exclude group: "com.fasterxml.jackson.core", module: "jackson-annotations"
+        exclude group: "com.fasterxml.jackson.core", module: "jackson-core"
+        exclude group: "com.fasterxml.jackson.core", module: "jackson-databind"
+        exclude group: "com.fasterxml.jackson.datatype", module: "jackson-datatype-jdk8"
+        exclude group: "javax.validation", module: "validation-api"
```

Comment:
They should not be excluded, or 2.6.7 of the same libraries should be included explicitly (like you did in https://github.com/civitaspo/embulk-filter-to_json/pull/8/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R40-R47).

If `embulkPluginRuntime.lockfile` does not include Jackson, it won't work with Embulk v0.11.

Reply:
As said, they are needed. `embulk-util-config` uses plugin's Jackson in it after v0.10.32. If the plugin does not include Jackson, its config processing won't work.